javascript/features/player_commands/commands/spawn_weapons.js

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 Las Venturas Playground. All rights reserved.
+// Use of this source code is governed by the MIT license, a copy of which can
+// be found in the LICENSE file.
+
+import CommandBuilder from 'components/command_manager/command_builder.js';
+import { PlayerCommand } from 'features/player_commands/player_command.js';
+import { WeaponData } from 'features/player_commands/weapon_data.js';
+
+// Using the spawnweapons command is a bit more expensive than using the ammunation.
+const ExtraPriceFactor = 1.5;
+
+// Command: spawnweapons [weaponId] [multipler]
+export default class SpawnWeapons extends PlayerCommand {
+    get name() { return 'spawnweapons'; }
+
+    build(commandBuilder) {
+        commandBuilder
+            .parameters([{ name: 'weapon', type: CommandBuilder.NUMBER_PARAMETER, optional: false },
+            { name: 'multiplier', type: CommandBuilder.NUMBER_PARAMETER, optional: false }])
+            .build(SpawnWeapons.prototype.onSpawnWeaponsCommand.bind(this));
+    }
+
+    buildAdmin(commandBuilder) {
```

Comment:
Let's go with this for now, but I'll have a think about how we could remove the duplication.